Dockerfile

```diff
@@ -1,9 +1,11 @@
 ARG APP_INSIGHTS_AGENT_VERSION=3.2.6
 
 # Application image
-
 FROM hmctspublic.azurecr.io/base/java:17-distroless
 
+# Change to non-root privilege
+USER hmcts
```

Comment:
it is already running as non-root, this doesn't do anything:
https://github.com/hmcts/cnp-java-base/blob/master/distroless/Dockerfile#L49